client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -189,6 +190,15 @@ trait Client[F[_]] {
 
   @deprecated("Use expect", "0.14")
   def prepAs[T](req: F[Request[F]])(implicit d: EntityDecoder[F, T]): F[T]
+
+  /**
+    * Translates the effect type of this client from F to G
+    */
+  def imapK[G[_]: Defer: Bracket[?[_], Throwable]](fk: F ~> G)(gK: G ~> F)(
```

Comment:
Hmm.  This shouldn't get a cats-tagless instance, which was the `imapK` concern on `Kleisli`, but I wonder whether we should just call this `translate` for consistency with the server.  Thoughts?

Reply:
Yeah, I think I agree - have changed it.